Sources/PrivoSDK/api/Rest.swift

```diff
@@ -74,6 +74,7 @@ class Rest: Restable {
     private static let storageComponent = "storage"
     private static let putComponent = "put"
     private static let sessionID = "session_id"
+    private static let acceptableStatusCodes: Set<Int> = [200, 204, 205]
     private static let emptyResponsesCodes: Set<Int> = .init([200,204,205])
```

Comment:
do we need 2 of the same sets? I mean acceptableStatusCodes and emptyResponsesCodes